tests/unittest/test_models.py

```diff
@@ -207,3 +206,68 @@ def test_big_rnn_model_share_params():
     logits, hidden = eval_model(x, hidden)
     assert logits.shape == (seq_len, batch_size, vocab_size)
     mx.nd.waitall()
+
+def test_weight_drop():
```

Comment:
Perhaps we also need to test whether the gradient is correct. We can do it by asserting the new parameter.

Reply:
added